src/components/PersonList/PersonList.tsx

```diff
@@ -0,0 +1,94 @@
+import { FC } from 'react';
+import classNames from 'classnames';
+import { useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { PersonLink } from '../PersonLink/PersonLink';
+
+type Props = {
+  persons: null | Person[],
+};
+
+export const PersonList: FC<Props> = ({ persons }) => {
+  const { slug } = useParams();
+
+  if (!persons) {
+    return (
+      <p data-cy="noPeopleMessage">
+        {persons === null
+          ? ''
+          : 'There are no people on the server'}
+      </p>
+    );
+  }
+
+  return (
+    <table
+      data-cy="peopleTable"
+      className="table is-striped is-hoverable is-narrow is-fullwidth"
+    >
+      <thead>
+        <tr>
+          <th>Name</th>
+          <th>Sex</th>
+          <th>Born</th>
+          <th>Died</th>
+          <th>Mother</th>
+          <th>Father</th>
```

Comment:
We can cheat a little and put the column names in an array outside the component and `.map` on it in the same way as we do with the `persons` array.